static/usage/v8/theming/class-dark-mode/javascript/index_ts.md

```diff
@@ -0,0 +1,35 @@
+```ts
+import { defineCustomElements } from '@ionic/core/loader';
+
+/* Core CSS required for Ionic components to work properly */
+import '@ionic/core/css/core.css';
+
+/* Basic CSS for apps built with Ionic */
+import '@ionic/core/css/normalize.css';
+import '@ionic/core/css/structure.css';
+import '@ionic/core/css/typography.css';
+
+/* Optional CSS utils that can be commented out */
+import '@ionic/core/css/padding.css';
+import '@ionic/core/css/float-elements.css';
+import '@ionic/core/css/text-alignment.css';
+import '@ionic/core/css/text-transformation.css';
+import '@ionic/core/css/flex-utils.css';
+import '@ionic/core/css/display.css';
+
+/**
+ * Ionic Dark Theme
+ * -----------------------------------------------------
+ * For more information, please see:
+ * https://ionicframework.com/docs/theming/dark-mode
+ */
+
+// import '@ionic/core/css/themes/dark.always.css';
+import '@ionic/core/css/themes/dark.class.css';
+// import '@ionic/core/css/themes/dark.system.css';
```

Comment:
Suggestion: Remove the commented out imports since this file targets the class dark mode. The same goes for the other class dark mode examples.